custom_components/tuya_local/devices/makegood_switch_dual_timer-energy.yaml

```diff
@@ -0,0 +1,87 @@
+name: Dual switch with timers and power monitoring
+products:
+  - id: XdGCESvXTI5Mvfx2
+    name: MakeGood 2 Gang Switch (MG-MD04W)
+primary_entity:
+  entity: switch
+  name: Switch 1
+  dps:
+    - id: 1
+      type: boolean
+      name: switch
+secondary_entities:
+  - entity: switch
+    name: Switch 2
+    dps:
+      - id: 2
+        type: boolean
+        name: switch
+  - entity: number
+    name: Timer 1
+    icon: "mdi:timer"
+    category: config
+    dps:
+      - id: 7
+        type: integer
+        name: value
+        range:
+          min: 0
+          max: 86400
+        mapping:
+          - scale: 60
+            step: 60
+        unit: min
+  - entity: number
+    name: Timer 2
+    icon: "mdi:timer"
+    category: config
+    dps:
+      - id: 8
+        type: integer
+        name: value
+        range:
+          min: 0
+          max: 86400
+        mapping:
+          - scale: 60
+            step: 60
+        unit: min
+  - entity: sensor
+    name: Current
```

Comment:
Remove the name when it matches the class (same for power and voltage).

This is a new feature in the next release, so you'll need to be using the main branch HEAD for testing.

Reply:
There is a difference in case `Current` vs. `current`. The name should be in `Title` case. Is this handled?

Reply:
If the name is omitted, then Home Assistant will use its standard translations (the translations take care of capitalising the name).

Reply:
I do love the structure of HA!

Reply:
I copied those lines from other devices so there will be a lot to update.
While updating, the yaml files could do with a standard structure such as:
<manufacturer>_<type>_<subtype>_<features>.yaml

Reply:
Maybe you should be marking these as resolved??

Reply:
You must have copied the file before Sunday, as the change was made here 927f7c779b

Renaming config files is a lot more complicated, as the names are stored in the config for devices that have been set up.